attelo/decoding/control.py

```diff
@@ -60,10 +60,14 @@ def _predict_relate(dpack, models):
     and an list of labels
     """
     if models.relate == 'oracle':
-        idxes = dpack.target
+        unrelated = dpack.label_number(UNRELATED)
```

Comment:
OK! Now updated, hopefully clearer now.
I'll defer the decision on probs/scores etc later, but I've taken the liberty of renaming prob_distrib to candidates.  Will push this in an hour or so.